lib/osut/utils.rb

```diff
@@ -61,76 +61,75 @@ module OSut
   #   - thickness                  :    0.1 m
   #   - thermal conductivity (k  ) :    0.1 W/m.K
   #   - density              (rho) :    0.1 kg/m3
-  #   - specific heat        (cp ) : 1400.0 J/kg.K
+  #   - specific heat        (cp ) : 1400.0 J/kg•K
   #
   #   https://s3.amazonaws.com/openstudio-sdk-documentation/cpp/
   #   OpenStudio-3.6.1-doc/model/html/
   #   classopenstudio_1_1model_1_1_standard_opaque_material.html
   #
   # ... apart from surface roughness, rarely would these material properties be
-  # suitable - and are therefore explicitely set below. On roughness:
+  # suitable - and are therefore explicitly set below. On roughness:
   #   - "Very Rough"    : stucco
   #   - "Rough"	        : brick
   #   - "Medium Rough"  : concrete
   #   - "Medium Smooth" : clear pine
   #   - "Smooth"        : smooth plaster
   #   - "Very Smooth"   : glass
 
-  # thermal mass categories (e.g. exterior cladding, interior finish, framing)
+  # Thermal mass categories (e.g. exterior cladding, interior finish, framing).
   @@mass = [
       :none, # token for 'no user selection', resort to defaults
      :light, # e.g. 16mm drywall interior
     :medium, # e.g. 100mm brick cladding
      :heavy  # e.g. 200mm poured concrete
   ].freeze
 
-  # basic materials (StandardOpaqueMaterials only)
+  # Basic materials (StandardOpaqueMaterials only).
   @@mats = {
+    material: {}, # generic, e.g. lightweight cladding over furring, fibreboard
```

Comment:
A generic (default) material is retained to cover all lightweight cladding and sheathing needs. When assemblies are adequately insulated, there's no point dragging different fibre-based materials with subtle differences in properties).